### PR TITLE
feat: hide stderr on git push with auth in URL #87

### PR DESCRIPTION
**Pull request by janl**

### CODE REVIEW

File: upload.js [hunk 1]
```diff
@@ -49,8 +50,16 @@ module.exports = function upload () {
...
 
+  const err = fs.createWriteStream('gk-lockfile-git-push.err')
+
   exec(`git remote add gk-origin ${remote}`)
-  exec(`git push gk-origin HEAD:${info.branchName}`)
+  exec(`git push gk-origin HEAD:${info.branchName}`, {
+    stdio: [
+      'pipe',
+      'pipe',
+      err
```
Comment (gr2m) [inline upload.js]: are you sure this works with a write stream? This fails for me

```js
exec(`ls`, {
  stdio: [
    'pipe',
    'pipe',
    fs.createWriteStream('err.our')
  ]
})
// TypeError [ERR_INVALID_OPT_VALUE]: The value "WriteStream {...}" is invalid for option "stdio"
```

This works

```js
exec(`ls`, {
  stdio: [
    'pipe',
    'pipe',
    fs.openSync('err.out', 'w')
  ]
})
```

Comment (janl) [reply upload.js]: I swear I tried this in a repl, but I can’t reproduce it. Can you send a patch? <3

Comment (gr2m) [reply upload.js]: Shall I amend the PR?

Comment (janl) [reply upload.js]: be my guest :)

Comment (gr2m) [reply upload.js]: there you go f8fbbb4, make sure to squash & merge

**Pull request merged**